.github/workflows/pizza.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Pizza Action
-        uses: open-sauced/pizza-action@v2.3.0
+        uses: open-sauced/pizza-action@v2
```

Change back per ^ conversation

```suggestion
        uses: open-sauced/pizza-action@v2.4.0
```